core/src/main/resources/reference.conf

```diff
@@ -1,16 +1,5 @@
 # Copyright 2016 Dennis Vriend
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# Copyright (C) 2019 - 2021 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
2022, in several places

Reply:
Yeah, all over this project, didn't want to change that in the same PR, I'll follow up with that

Reply:
Changed my mind, bumped in [f22c3eb](https://github.com/akka/akka-persistence-jdbc/pull/677/commits/f22c3eb0032324756930fc48e3d06f9a582d70dc)